Gordon360/Static Classes/Helpers.cs

```diff
@@ -17,5 +17,16 @@ public static string GetCurrentSession(CCTContext context)
         {
             return context.CM_SESSION_MSTR.Where(s => DateTime.Now > s.SESS_BEGN_DTE).OrderByDescending(s => s.SESS_BEGN_DTE).Select(s => s.SESS_CDE).First();
         }
+
+        /// <summary>
+        /// Helper method that casts given date time to type UTC without changing the time
+        /// </summary>
+        /// <param name="dateTime"></param>
+        /// <returns>dateTime with type UTC, if dateTime is null, will return DateTime min value</returns>
+        public static DateTime FormatDateTimeToUtc(DateTime? dateTime)
+        {
+            if (dateTime is null) return DateTime.MinValue;
```

Comment:
datetime can't be cast as null ;_; (I didn't make the rules) The only way that it can appear as null is if I don't pass in anything. If I explicitly cast null to datetime i.e `(DateTime) null`, I get DateTime.MinValue anyways. So to lessen the confusion I just returned the min value.

Reply:
Well yes, `DateTime` is a value type so it can't be null. But you can make the method return a nullable `DateTime?` in which case you can return null.

Reply:
Yeah it’s a limitation of the compiler, you can’t use the `Thing?.Property` syntax in certain lambda expressions. Presumably that's related to the call site of this method, not this method itself. You'd just need to handle the null return differently.

Reply:
We're having this discussion offline, perhaps a second function that is nullable is required to handle cases of nullable and non-nullable?

Reply:
Yeah, you could overload it for nullable and non-nullable DateTimes.

Also, you could make this an extension method instead of a static helper method. Might be slightly more ergonomic to do e.g. `StartTime?.SpecifyUTC()`

Reply:
I'm not sure if it would be more ergonomic 
1) not sure how I would do it, since StartTime is a datetime object. I don't know if I can add methods to System.Datetime's object
2) if I had that as a class method, I would need it for every single viewmodel
3) unless you're thinking about `DateTime.ToUniversalTime` which unfortunately, as far as I'm aware, thinks that unspecified times are local times. Thus it would add an additional 5 hours.

Reply:
Welcome to the magical world of extension methods: https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods

Reply:
this feels illegal :0
I'll try it out